templates/redhat/dhcpd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=DHCPv4 Server Daemon
+Documentation=man:dhcpd(8) man:dhcpd.conf(5)
+Wants=network-online.target
+After=network-online.target
+After=time-sync.target
+
+[Service]
+Type=notify
+ExecStart=/usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid  <%= @dhcp_interfaces.join(' ') %>
```

Comment:
Continuing discussion from https://github.com/theforeman/puppet-dhcp/pull/67#discussion_r59350684

You should be able to use a systemd drop-in snippet instead of including the entire unit file. You'd create a `/etc/systemd/system/dhcpd.service.d/execstart.conf` file and only include the changes.